__tests__/commands/install/integration.js

```diff
@@ -208,13 +208,17 @@ test.concurrent('hoisting should factor ignored dependencies', async () => {
 });
 
 test.concurrent('--production flag ignores dev dependencies', () => {
-  return runInstall({production: true}, 'install-production', async (config) => {
+  return runInstall({production: true}, 'install-production-without-dev', async (config) => {
```

Comment:
Basic tests are good :) If they break, we don't have to debug which part of the test is failing